src/main/java/org/ecocean/servlet/GetCurrentIAInfo.java

```diff
@@ -110,22 +110,44 @@ private JSONObject getIAJSONForAnnotation(Shepherd myShepherd, Annotation ann) {
             annIA.put("assetDetectionStatus", ma.getDetectionStatus());
             annIA.put("assetWebURL", Util.scrubURL(ma.webURL()));
             
-             List<Task> rootTasks = ann.getRootIATasks(myShepherd);
-            if (rootTasks!=null&&!rootTasks.isEmpty()) {
+            //List<Task> rootTasks = ann.getRootIATasks(myShepherd);
```

Comment:
Commenting out will become cruft, decrease readability, and potentially make merge conflicts more difficult to resolve.

Let's remove them when we can.

Reply:
Removed.